src/modules/books.js

```diff
@@ -0,0 +1,25 @@
+const books = () => {
+  const formData = document.querySelector('.form-control');
+  const bookName = document.getElementById('name');
+  const bookAuthor = document.getElementById('author');
+  const bookArray = [] || JSON.parse(localStorage.getItem('story'));
+  // if (localStorage.getItem('story')) {
+  //   bookArray = JSON.parse(localStorage.getItem('story'));
+  // }
+
+  formData.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const { length } = bookArray;
+    let id = 0;
+    if (length > 0) {
+      id = length;
+    }
+    const book = { title: bookName.value, author: bookAuthor.value, id };
+    const data = JSON.parse(localStorage.getItem('story'));
+    data.push(book);
```

Comment:
- [ ] When the app is opened for the first time in the browser, the `story` item is still not registered in the localStorage. Thus, when trying to create a new book, the error below is encountered.

![image](https://user-images.githubusercontent.com/45002654/236233052-424e6a21-b5d2-45b2-b682-c2f0479ee6b8.png)

We cannot push to an object that is undefined. I can see you already have bookArray at line 5. Please try and work with that.